src/App.tsx

```diff
@@ -1,27 +1,58 @@
-import React from 'react';
 import './App.scss';
+import { useState } from 'react';
 import { GoodsList } from './GoodsList';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { getAll, get5First, getRedGoods } from './api/goods';
+import { Good } from './types/Good';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <h1>Dynamic list of Goods</h1>
+export const App: React.FC = () => {
+  const [goods, setGoods] = useState<Good[]>([]);
 
-    <button type="button" data-cy="all-button">
-      Load all goods
-    </button>
+  const handleLoaderOfAllGoods = () => {
+    getAll()
+      .then(setGoods);
+  };
 
-    <button type="button" data-cy="first-five-button">
-      Load 5 first goods
-    </button>
+  const handleLoaderOfFiveFirstGoods = () => {
+    get5First()
+      .then(setGoods);
+  };
 
-    <button type="button" data-cy="red-button">
-      Load red goods
-    </button>
+  const handleLoaderOfRedGoods = () => {
+    getRedGoods()
+      .then(setGoods);
+  };
```

Comment:
What do you think about creating one callback that accepts the function?
Seems like your functions do the same:
1. calling the function which returns the promise
2. pass `setGoods` function inside `then` method